src/languages/es.js

```diff
@@ -207,6 +207,9 @@ export default {
         notifications: 'Notificaciones',
         receiveRelevantFeatureUpdatesAndExpensifyNews: 'Recibir noticias sobre Expensify y actualizaciones del producto',
         priorityMode: 'Modo Prioridad',
+
+        // TODO
+        language: '????',
```

Comment:
Hey @iwiznia! What would be the correct translation for the "Language" in the context of the language picker?

Reply:
I'd say it's `Idioma`. Next time, feel free to jump in #espanol and ask there 😉

Reply:
Gotcha, thanks a lot!